File: coding/Queue/python/ArrayQueue.py
class ArrayQueue:
    def __init__(self):
        self.array = []

    def isEmpty(self):
        if len(self.array) == 0:
            return True
        else:
            return False

    def enQueue(self, value):
        self.array.append(value)

    def deQueue(self):
        return self.array.pop(0)

File: coding/Queue/Java/src/com/Queue/ArrayQueue.java
package com.Queue;
import com.Array.LVector;

public class ArrayQueue {
    private LVector array;

    public ArrayQueue(){
        array = new LVector(10);
    }

    public boolean isEmpty(){
        return array.IsEmpty();
    }

    public void enQueue(int value){
        array.Push(value);
    }

    public int deQueue(){
        int value = array.GetValueAt(0);
        array.Delete(0);

        return value;
    }
}
